mmedit/datasets/pipelines/augmentation.py

```diff
@@ -1031,3 +1031,86 @@ def __repr__(self):
         repr_str = self.__class__.__name__
         repr_str += (f'(keys={self.keys})')
         return repr_str
+
+
+@PIPELINES.register_module()
+class CopyValueFromKey:
```

Comment:
The name `CopyValueFromKey` is okayish. Does the name come from other mm-repos? If it is completely new, how about a shorter name e.g. `Clone/Copy/CopyValues`?